### PR TITLE
Add events dates in email notifications

### PR DESCRIPTION
**Pull request by sponkrashin**
Issue #402

### CODE REVIEW

File: server/Arcadia.Assistant.Calendar.Notifications/EventAssignedToApproverPushNotificationActor.cs [hunk 1]
```diff
@@ -105,7 +105,9 @@ private PushNotification CreatePushNotification(CalendarEventAssignedWithAdditio
             var content = new PushNotificationContent
             {
                 Title = this.pushNotificationConfig.Title,
-                Body = string.Format(this.pushNotificationConfig.Body, message.Event.Type, message.Owner.Name),
+                Body = this.pushNotificationConfig.Body
```
Comment (sponkrashin) [inline server/Arcadia.Assistant.Calendar.Notifications/EventAssignedToApproverPushNotificationActor.cs]: @santee, do we also need dates in push notifications?

Comment (santee) [reply server/Arcadia.Assistant.Calendar.Notifications/EventAssignedToApproverPushNotificationActor.cs]: nah

File: server/Arcadia.Assistant.Server.Console/appsettings.json [hunk 2]
```diff
@@ -35,36 +35,36 @@
...
         "NotificationSender": "",
         "Subject": "Approved sick leave",
-        "Body": "Dear Sir/Madam,\nNotify you, that employee {0} is on sick leave starting {1}.\nYours faithfully,\nTesting team"
+        "Body": "Dear Sir/Madam,\nNotify you, that employee {employee} is on sick leave starting {startDate}.\nYours faithfully,\nTesting team"
       },
       "EventAssignedToApprover": {
         "NotificationSender": "", 
         "Subject": "Pending calendar event",
-        "Body": "Dear Sir/Madam,\nNotify you, that {0} event of employee {1} needs your approval.\nYours faithfully,\nTesting team"
+        "Body": "Dear Sir/Madam,\nNotify you, that {eventType} {dates} of employee {employee} needs your approval.\nYours faithfully,\nTesting team"
```
Comment (santee) [inline server/Arcadia.Assistant.Server.Console/appsettings.json]: put dates in () :)

Comment (sponkrashin) [reply server/Arcadia.Assistant.Server.Console/appsettings.json]: Done.

**Pull request merged**